static/examples.json

```diff
@@ -1237,7 +1241,8 @@
     ],
     "tags": [
       "urls"
-    ]
+    ],
+    "schema": "listing_url"
```

Comment:
nit: should sort this back to where it was previously. WIll be a bit annoying to parse through

Let's also sort tags higher than the actual data while we're at it

Reply:
ya fair done